### DISCUSSION TITLE
Suggestion/Question about putting italics codes on both lines of a subtitle

### DISCUSSION THREAD

**Discussion by JVPCorreia** [SubtitleEdit/subtitleedit#6954] [Ideas]
Hello!

So, I've been using SE for some years now and have appreciated so many of the improvements you all have been doing.

However, if I could suggest something - which I'm not sure is possible. Is there way to put the complete italics code in each line of the subtitle for future versions?
For example: 
<i>Example, example</i>
<i>example example.</i>

Instead of:
<i>Example, example
example example.</i>

I don't know if it's possible to change that in some settings in the current version of SE. If it is, I couldn't find it.

This type of italics is being requested more and more since 2020 for new projects and formats, such as SLT. 
I've been doing it manually, selecting the line and pushing ctrl+I for all italics lines. For some projects, that's quite time-consuming.

If anyone knows something about this and can shine a light on this, I'd appreciate it!

Thanks!!

**Comment by niksedk** [comment]
You can try the plugin "Italic each line" (go to File - Plugins):
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/c198a7ed-12f0-4b15-aa62-1b7982132b4c)


To do the opposite (if needed), use Fix Common Errors:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/b37b62df-ca61-43ce-8ce8-ee06fb95771d)

**Comment by JVPCorreia** [comment]
Thanks!!
Installed and tested. Works perfectly!

Thanks again, and sorry for not doing a better job looking for a solution before opening a discussion.
That was my first plugin, and now this opened a whole new world for me!
